tests/crypto_aesctr/generate_test_vectors.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python3
+
+""" Generate AES-CTR test vectors with an independent implementation. """
+
+import binascii
+
+import Crypto.Util.Counter
```

Comment:
In case you're wondering, the blank line separates `binascii`, which is part of the standard library, from the `Crypto` modules, which is a separate `pip install`.